defaults/main.yml

```diff
@@ -24,10 +24,10 @@ atom_csrf_protection: "no"
 
 #
 # Choose php version to use
-#   - For RH/Centos 70,71,72 and 73 are available. Default is 72
-#   - Ubuntu 14.04 uses php 5, 16.04 7.0 and 18.04 7.2
+#   - For RH/Centos 70,71,72,73 and 7.4 are available. Default is 72
```

Comment:
(Kind of nitpicky, but) should it be "For RH/Centos 70,71,72,73 and 74 ...." instead of "For RH/Centos 70,71,72,73 and 7.4 ...."